chain/client/src/client.rs

```diff
@@ -803,8 +803,27 @@ impl Client {
                 )?;
 
                 match process_result {
-                    ProcessPartialEncodedChunkResult::Known => Ok(vec![]),
-                    ProcessPartialEncodedChunkResult::HaveAllPartsAndReceipts(prev_block_hash) => {
+                    ProcessPartialEncodedChunkResult::Known(demurs_remain) => {
+                        if demurs_remain {
```

Comment:
do we actually need to do this? if we have remaining demurs the request is not removed from the request pool, which means that it will be retried later.

Reply:
Yeah, good point. Removed in https://github.com/near/nearcore/pull/3657/commits/30be273f0617e4267878988e76573be9ce49516e